packages/common/src/ui/index.ts

```diff
@@ -6,5 +6,4 @@ export * from './components';
 export * from './forms';
 export * from './icons';
 export * from './layout';
-export * from './notifications';
```

Comment:
The naming of Alert*Modal* made me move it to the modal folder... though the terminology between modals, dialogs and alerts isn't quite clear and distinct!  🤔

Reply:
I'm thinking about renaming all the dialog bits to 'modal' and be done with it. I think of dialog as a popup thingy and a modal as a blocking popup thingy - you can't continue until you interact with it.

Since we are only using blocking alert windows, they are really all _modals_ and we might as well just call them that!